crates/project/src/toolchain_store.rs

```diff
@@ -345,9 +350,11 @@ impl RemoteToolchainStore {
             Some(())
         })
     }
+
     pub(crate) fn list_toolchains(
         &self,
         worktree_id: WorktreeId,
+        _project_env: Option<HashMap<String, String>>, // TODO: use project_env on remote somehow?
```

Comment:
We should probably update `zed.proto` to pass project env, kinda like we do for tasks in https://github.com/zed-industries/zed/blob/3856599853b2a8aab15728847c737d13b08c3bfe/crates/proto/proto/zed.proto#L2187

Reply:
It's just weird, like, the environment is already from the remote, no? Why would I roundtrip it to the host and back?

Reply:
Anyways, let's do this for now and leave this to deal with for remoting people for later

Reply:
Scratch that, I've come up with a better solution

Reply:
It is better indeed :+1: